custom_components/hyxi_cloud/coordinator.py

```
@@ -112,11 +117,32 @@ async def _async_update_data(self):
+                    await self._async_sync_device_metadata(cached_devices)
+                    return cached_devices
+                else:
+                    _LOGGER.warning(
+                        "HYXI Cloud returned success, but no plants or devices were found. "
+                        "If your developer email differs from your app email, you must share your Plant "
+                        "from the app to the developer email first."
+                    )
+            else:
+                try:
+                    await self.device_store.async_save(devices)
+                except Exception as save_err:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)